audit/audit_dummy.go

```diff
@@ -0,0 +1,40 @@
+package audit
+
+import (
+	"encoding/json"
+	"github.com/ONSdigital/go-ns/log"
+)
+
+type DummyProducer struct {
```

Comment:
where is this file used? has this file been superceded by OutboundProducerMock?

Reply:
Didn't know we had one - this is just temp and will be delete shorty.